recipes/onetbb/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "2022.1.0":
+    url: "https://github.com/uxlfoundation/oneTBB/archive/v2022.1.0.tar.gz"
```

Comment:
The naming change is valid, the repository has been renamed.

Unless we need to modify the recipe, we probably don't wan't to change the old URLs to keep the old revisions intact